internal/webhook/webhook.go

```diff
@@ -23,35 +25,44 @@ type Handler interface {
 type Webhook struct {
 	client.Client
 	Config    *config.Config
-	Providers []Provider
+	Providers map[string][]gitprovider.Provider
 }
 
 func New(c *config.Config) *Webhook {
 	return &Webhook{
-		Config: c,
+		Config:    c,
+		Providers: make(map[string][]gitprovider.Provider),
 	}
 }
 
 type Provider interface {
```

Comment:
Is this interface still needed?

Reply:
This was the old one, good catch. I'm deleting it